pkg/administration/events/codes/codes.go

```diff
@@ -4,6 +4,10 @@ package codes
 // The error codes may be used for filtering within logs as well as rendering
 // specific hints for the related administration events.
 const (
+	// API Token codes.
+	TokenCreated = "token-created"
+	TokenExpired = "token-expired"
```

Comment:
If these are API (long-lived) token specific, I'd have this expressed in both variables and contents.